trustpoint/devices/views.py

```diff
@@ -1014,9 +1030,14 @@ def get_context_data(self, **kwargs: Any) -> dict[str, Any]:
         else:
             err_msg = 'Unsupported public key algorithm'
             raise ValueError(err_msg)
-        context['host'] = (
-            f'{self.request.META.get("REMOTE_ADDR", "127.0.0.1")}:{self.request.META.get("SERVER_PORT", "443")}'
-        )
+        try:
```

Comment:
Perhaps it would make sense to move this code to the `TlsSettings` model as `get_preferred_ipv4()` or similar.
That would prevent having the DB query code duplicated across multiple views.